config/meson.build

```
@@ -29,6 +29,14 @@ if fc_id == 'gcc'
     '-Wno-maybe-uninitialized',
     language: 'fortran',
   )
+  message('GCC version: ' + fc.version())
+  if fc.version().contains('9.')
```

https://mesonbuild.com/Reference-manual_elementary_[str](https://mesonbuild.com/Reference-manual_elementary_str.html#strversion_compare).html#strversion_compare
```suggestion
  if fc.version().version_compare('<10')
```